src/rules/no_var.rs

```diff
@@ -63,5 +81,32 @@ mod tests {
       r#"var someVar = "someString"; const c = "c"; let a = "a";"#,
       0,
     );
+
+    assert_lint_err!(
+      NoVar,
+      "var foo = 'bar';": [{
+        col: 0,
+        message: MESSAGE,
+      }],
+      "let foo = 'bar'; var i = 0;": [{
+        col: 17,
+        message: MESSAGE,
+      }],
+      "let foo = 'bar'; var i = 0; var x = 1;": [
+        {
+          col: 17,
+          message: MESSAGE,
+        },
+        {
+          col: 28,
+          message: MESSAGE,
+        }
+      ]
+    );
+  }
+
+  #[test]
+  fn no_var_valid() {
+    assert_lint_ok::<NoVar>(r#"let foo = 0; const bar = "bar""#);
   }
```

Comment:
Can you move `no_var_valid` function above the `no_var_invalid` function? Because all the tests for the rules follow this convention of order :)

Reply:
Yes, I moved it to top now